src/profile-logic/marker-schema.js

```diff
@@ -286,7 +285,7 @@ export function parseLabel(
   };
 }
 
-type LabelKey = 'tooltipLabel' | 'tableLabel' | 'chartLabel';
+type LabelKey = 'tooltipLabel' | 'tableLabel' | 'chartLabel' | 'copyLabel';
```

Comment:
so this new label key is specifically for getting the text to copy right? I guess it would be nice to file a bug for this in the backend as well.

Reply:
Yes, it is specifically to get the text to copy. The code in the PR doesn't support getting a 'copyLabel' template string from the marker schema. In the future we could easily make it so that the implementation uses a copyLabel template from the schema, and fallbacks to `{marker.name} - {tableLabel}` if there is none. For now I implemented the "fallback" directly, as I couldn't think of a use case where we would want the copy string to differ significantly from the table label. Unless we change this in the implementation to actually support a copyLabel template, I don't think there's anything to file a bugzilla bug for.